### DISCUSSION TITLE
Delete/Reset goal conversion hits

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#186] [Anything Else]
"When you set up goals, you usually test them yourself so the first hits for the goals are fake. I need a way to wipe existing hits to make sure goals are unbiased."

(feedback via email)

**Comment by rasben** [comment]
+1! :)

**Reply by deploytoon** [reply]
+1

**Reply by rubzie** [reply]
Would love this indeed!

**Reply by rubzie** [reply]
I tried removing the goal altogether, but readding it also readds the conversions :o

**Comment by mrgiao** [comment]
### Suggested solution
* Provide a reset action per row (goals and custom props).
* Provide relevant information before committing the destructive action.
* Link to more in-depth information at Docs.
* Allow user to actually reset stats.

![2022-01-14_12-11-13](https://user-images.githubusercontent.com/4223897/149506364-01d50e3b-f7fd-4e5b-ba62-99bf65229a66.png)

**Reply by ukutaht** [reply]
Looks great!

I think the reset button should be in site settings rather than taking up space in the dashboard.

**Reply by mrgiao** [reply]
Sure. In an initial version maybe resetting the goal is enough.

![chrome_2022-01-14_18-24-17](https://user-images.githubusercontent.com/4223897/149558257-a44df2d0-4579-46f4-bb05-4030c8afdb27.png)

**Reply by ukutaht** [reply]
That looks great to me!

**Comment by Sadashii** [comment]
Is there currently any way to do this manually since the feature is yet to be added?

**Comment by RafidMuhymin** [comment]
Any progress on this feature?

**Comment by alamenai** [comment]
What if we send the user's email/username with the custom properties to exclude our faking tests?
